Dockerfiles/python/.devcontainer/Dockerfile

```diff
@@ -52,13 +52,16 @@ ADD --chown=4000 https://raw.githubusercontent.com/eclipse-velocitas/vehicle-app
 ADD --chown=4000 https://raw.githubusercontent.com/eclipse-velocitas/vehicle-app-python-template/main/app/requirements-links.txt /requirements/
 ADD --chown=4000 https://raw.githubusercontent.com/eclipse-velocitas/vehicle-app-python-template/main/app/requirements-app.txt /requirements/
 ADD --chown=4000 https://raw.githubusercontent.com/eclipse-velocitas/vehicle-app-python-template/main/app/tests/requirements-tests.txt /requirements/
+COPY --chown=4000 ./common/requirements-common.txt /requirements/
 
 WORKDIR /requirements
 # Always install requirements under vscode user to avoid issue with installation of packages
-RUN pip3 install -r requirements-dev.txt
-RUN pip3 install -r requirements-links.txt
-RUN pip3 install -r requirements-app.txt
-RUN pip3 install -r requirements-tests.txt
+RUN pip3 install -r requirements-dev.txt && \
```

Comment:
Kind of optional: 
I just noticed this now but we should remove these requirements in the end since they will be available in the final image. Could you add a final sudo rm -rf /requirements ? Maybe you should not change the workdir to /requirements in that case. Same also applies for the /scripts directory or however we decide to call it.

Could also be handled in a different task/PR though.

Reply:
- Shouldn't we put them then into a subfolder of /tmp?
- What about the scripts? Are they used later or could also be removed?